<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>414</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QFrame" name="frame">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>File Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit">
              <property name="styleSheet">
               <string notr="true">color: rgb(0, 0, 0);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Column Name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Max # of Column</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLineEdit" name="lineEdit_2">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 0);</string>
                </property>
                <property name="placeholderText">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_3">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 0);</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="font">
             <font>
              <pointsize>30</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(40, 167, 69);</string>
            </property>
            <property name="text">
             <string>Extract!</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QTextBrowser" name="debugTextBrowser">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>633</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>debugTextBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>writeDocSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>598</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>631</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>returnedPressedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>browseSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>567</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_2</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>columnNameSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_3</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>maxColumnSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browseSlot()</slot>
  <slot>writeDocSlot()</slot>
  <slot>returnedPressedSlot()</slot>
  <slot>columnNameSlot()</slot>
  <slot>maxColumnSlot()</slot>
 </slots>
</ui>
